portal/public/index.html

```diff
@@ -145,20 +145,38 @@
     </style>
 </head>
 <body>
-
-    <div class="mobile">
-        <div class="mobile-content">
-          <img src="/images/surflogo@3x.png">
-          <p>Het zoekportaal is gedurende de pilot niet te gebruiken op kleine schermen. Probeer je device te draaien of gebruik een computer.</p>
-          <p>During the pilot the search portal can't be used on small screens. Try to rotate your device or use a desktop computer.</p>
-        </div>
-    </div>
-
-    <div id="__nuxt">
-        <div id="nuxt-loading" aria-live="polite" role="status">
-            <div>Loading...</div>
-        </div>
-    </div>
-
+    <noscript>
+     <strong>Het zoekportaal heeft JavaScript nodig om te functioneren. Zet JavaScript aan in de browser.</strong>
+    </noscript>
+
+    <script type="text/javascript">
+        const isIE = /*@cc_on!@*/false || !!document.documentMode;
+        const isEdge = !isIE && !!window.StyleMedia;
+
+        if (isEdge || isIE) {
+          document.write("" +
```

Comment:
Hm, kan dit niet op een andere manier?

Reply:
Optie: vervang tekst in de <p> in plaats van alle elements renderen met write.

Reply:
ik kan ook de elementen maken met js. maar we willen juist die nuxt id niet hebben, zodat vue niet wordt aangezwengeld

Reply:
Ik zou in `src/main.js` niet Vue mounten als het IE is. Ipv daarvan die IE melding tonen.

Reply:
En die main.js kun je niet aanpassen. die wordt door nuxt gegenereerd.